README.md

```diff
@@ -1,33 +1,34 @@
-# ShellJS - Unix shell commands for Node.js
+# ShellJS - Unix shell commands for NodeJS
 
 [![Gitter](https://img.shields.io/gitter/room/nwjs/nw.js.svg?style=flat-square)](https://gitter.im/shelljs/shelljs?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 [![Travis](https://img.shields.io/travis/shelljs/shelljs/master.svg?style=flat-square&label=unix)](https://travis-ci.org/shelljs/shelljs)
 [![AppVeyor](https://img.shields.io/appveyor/ci/shelljs/shelljs/master.svg?style=flat-square&label=windows)](https://ci.appveyor.com/project/shelljs/shelljs/branch/master)
 [![npm version](https://img.shields.io/npm/v/shelljs.svg?style=flat-square)](https://www.npmjs.com/package/shelljs)
 [![npm downloads](https://img.shields.io/npm/dm/shelljs.svg?style=flat-square)](https://www.npmjs.com/package/shelljs)
 
-ShellJS is a portable **(Windows/Linux/OS X)** implementation of Unix shell commands on top of the
-Node.js API. You can use it to eliminate your shell script's dependency on Unix while still keeping
-its familiar and powerful commands. You can also install it globally so you can run it from outside
-Node projects - say goodbye to those gnarly Bash scripts!
+ShellJS is a portable **(Windows/Linux/OS X)** implementation of Unix shell
+commands on top of the Node.js API. You can use it to eliminate your shell
+script's dependency on Unix while still keeping its familiar and powerful
+commands. You can also install it globally so you can run it from outside Node
+projects - say goodbye to those gnarly Bash scripts!
 
 ShellJS is proudly tested on every node release since `v0.11`!
```

Comment:
That hasn't been merged to master branch yet, only dev. And I think that README change already exists in the dev branch.

We could just as well base this PR off dev, instead. No hurry.